File: Frontend/src/app/shared/api/models/event-create-dto.ts
/* tslint:disable */
/* eslint-disable */
import { EventCategory } from './event-category';
export interface EventCreateDto {
  category?: EventCategory;
  createdBy?: null | string;
  description?: null | string;
  end?: null | string;
  location?: null | string;
  start?: string;
  title?: null | string;
}

File: Webapi/Webapi/Models/Events/EventCreate.dto.cs
namespace Webapi.Models.Events;

#pragma warning disable CS8618 // data transfer object
public class EventCreateDto
{
  public string CreatedBy { get; set; }

  public string Title { get; set; }

  public string? Description { get; set; }

  public DateTime Start { get; set; }

  public DateTime? End { get; set; }

  public EventCategory? Category { get; set; }

  public string? Location { get; set; }
}